README.md

```diff
@@ -66,6 +66,31 @@ Assisted digital satisfaction surveys:
 * https://www.gov.uk/bank-holidays
 * https://www.gov.uk/when-do-the-clocks-change
 
+### Misc
+
+* http://www.gov.uk/school-term-holiday-dates
```

Comment:
If we know it, is it worth mentioning which view the pages use?

Reply:
Added ✅  

Partially explored how to add outputs to a Dev environment so comments easily identify which view is in use:
`<!-- <%= File.basename(__FILE__) %> -->`

Reply:
Is this valuable? If you run it locally/dev env you can watch what templates are called in the terminal already. The problem lies in the opposite direction when you need to know what urls a template generates and this doesn't support that.

Reply:
I think there is still value in this as you don't need to spin up the application to see what views a URL uses. I agree there's still work to be done to find out what URLs use which view - Owen's investigation into using Kibana looks to be promising for this.

Reply:
>Is this valuable? 

So the use case I found this useful was opening 30+ URLs at once to review and wanting to easily see what page I was looking at.   Is it easier to discover this in terminal in this use case?

> know what urls a template generates and this doesn't support that.

This was just an quick experiment - it would be nice to have this. 
(Wondering how that might work now)

Reply:
ah, I read "on a dev environment" as meaning you need to be running it anyway. This might make sense to scope to just dev/integration and implement - avoid giving millions of users useless bytes to collect. It's a nice idea.